src/lib/components/ILSMenu/ILSMenu.js

```diff
@@ -92,16 +92,14 @@ class ILSMenu extends Component {
             >
               <Container>
                 <Menu.Item header>
-                  <Overridable id="ILSMenu.Logo">
-                    <Link to="/">
-                      <Image
-                        src={uiConfig.LOGO_SRC}
-                        size="tiny"
-                        centered
-                        alt="Logo"
-                      />
-                    </Link>
-                  </Overridable>
+                  <Link to="/">
```

Comment:
I think it was made overridable on purpose by @ntarocco , because maybe you don't want always a picture, like on CDS books.